src/main/kotlin/com/openlattice/linking/controllers/LinkingFeedbackController.kt

```diff
@@ -65,7 +67,7 @@ constructor(
                     "Cannot submit feedback for less than 2 entities or if no positively linking entity is provided")
         }
 
-        if(!Sets.haveEmptyIntersection(feedback.link, feedback.unlink)) {
+        if( Collections.disjoint( feedback.link, feedback.unlink )) {
```

Comment:
it seems like the passing condition for this if statement changed from "have shared elements" to "don't have shared elements". Is that intended?

Reply:
+1